lighthouse-cli/test/fixtures/perf/fonts.html

```diff
@@ -6,14 +6,14 @@
         font-family: 'Lobster';
         font-style: normal;
         font-weight: 400;
-        src: local('Lobster'), url('./lobster-v20-latin-regular.eot?#iefix') format('eot'), url('./lobster-v20-latin-regular.woff2') format('woff2');
+        src: url('./lobster-v20-latin-regular.woff2') format('woff2');
```

Comment:
I added the local strings just to make sure the regex in getSheetsFontFaces works. Maybe we should keep it?

Reply:
Ah ok, thanks for the context I'll revert :)